shared/ui/Stream/PullRequestEditingComment.tsx

```diff
@@ -64,7 +64,7 @@ export const PullRequestEditingComment = styled((props: Props) => {
 	};
 
 	const handleCancelEdit = async () => {
-		if (text == null || text == undefined || text == props.text) {
+		if (text == null || text == undefined || text == props.text || text == props.pr.description) {
```

Comment:
IntelliJ recommended simplification (double equals covers null and undefind)

```suggestion
		if (text == null || text == props.text || text == props.pr.description) {
```